kong/cmd/utils/log.lua

```diff
@@ -24,6 +25,15 @@ function _M.set_lvl(lvl)
   end
 end
 
+function _M.disable()
+  old_lvl = log_lvl
```

Comment:
That would also defeat the purpose. The point is for kong 
stop/restart/reload to not print twice the entire configuration being 
parsed by conf_loader as it would look very confusing.

On 8/11/16 1:50 AM, Thijs Schreijer wrote:
>
> In kong/cmd/utils/log.lua 
> <https://github.com/Mashape/kong/pull/1486#discussion_r74387075>:
>
> > @@ -24,6 +25,15 @@ function _M.set_lvl(lvl)
> >    end
> >  end
> >
> > +function _M.disable()
> > +  old_lvl = log_lvl
>
> hence the message should be 'debug' level, so doesn't print unless 
> your debugging, and that's when you'd want to see it.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub 
> <https://github.com/Mashape/kong/pull/1486/files/4c1b1f6bb287c09c800f5413e654af5b6ccfa6ff#r74387075>, 
> or mute the thread 
> <https://github.com/notifications/unsubscribe-auth/ABEsNyZtl5FjMJ3jhoIEnPW-koCzhTkpks5qeuJggaJpZM4JhwBE>.
>